test/runner/browsers.js

```diff
@@ -1,25 +1,248 @@
-// This list is static, so no requests are required
-// in the command help menu.
-
-import { getBrowsers } from "./browserstack/api.js";
-
-export const browsers = [
-	"chrome",
-	"ie",
-	"firefox",
-	"edge",
-	"safari",
-	"opera",
-	"yandex",
-	"IE Mobile",
-	"Android Browser",
-	"Mobile Safari",
-	"jsdom"
-];
-
-// A function that can be used to update the above list.
-export async function getAvailableBrowsers() {
-	const browsers = await getBrowsers( { flat: true } );
-	const available = [ ...new Set( browsers.map( ( { browser } ) => browser ) ) ];
-	return available.concat( "jsdom" );
+import chalk from "chalk";
```

Comment:
This file is seen as new but most of it is the same as the previous `test/runner/browserstack/browsers.js` with minor modifications. It's unfortunate that we cannot see a better diff here.

One way around it would be to keep this file in `test/runner/browserstack/browsers.js` and only move it in a separate PR later; that should preserve the diff, even across the file rename later.

Do you think it's worth trying that?

Reply:
It is unfortunate, but I'm not sure it's worth all that. Can still diff locally.

Reply:
OK; this code is relatively new so it's not such big of a deal. If this affected the actual jQuery source code, it'd be worth to apply such a workaround IMO.